components/OptimusPage/WhatIsOlasOptimus.jsx

```diff
@@ -0,0 +1,139 @@
+import { SUB_HEADER_CLASS, SUB_HEADER_LG_CLASS } from 'common-util/classes';
+import SectionWrapper from 'components/Layout/SectionWrapper';
+import { Card } from 'components/ui/card';
+import Image from 'next/image';
+
+const CARD_BG =
+  'border-1.5 border-gray-200 rounded-2xl p-6 bg-gradient-to-t from-[#F2F4F7] to-white';
+
+const types = [
+  {
+    id: 'Trader',
+    src: '/images/predict-page/traders.png',
+    desc: 'Finds the best DeFi opportunities and makes smart investments.',
+    linkText: 'More about Optimus Trader agent',
+    link: '/services/babydegen',
+  },
+  {
+    id: 'Mech',
+    src: '/images/predict-page/mechs.png',
+    desc: 'Operating as Olas Mech agent economy, Mechs provide intelligence for Trader agents for managing DeFi assets.',
+    linkText: 'More about Olas Mech agent economy ',
+    link: '/agent-economies/mech',
+  },
+];
+
+const WhatIsOptimus = () => (
+  <div>
+    <h1 className={`${SUB_HEADER_CLASS} font-semibold text-left text-4xl mb-8`}>
+      What is Olas Optimus?
+    </h1>
+    <p className="mb-20">
+      Olas Optimus agent economy is a multi-agent system of autonomous AI agents
+      to streamline your DeFi experience. An autonomous AI agent intelligently
+      manages your assets on specific blockchain platforms. <br />
+      It strategically targets high-yield opportunities, optimizing your returns
+      within ecosystems like Optimism Mainnet and Base.
+    </p>
+  </div>
+);
+
+const AgentTypes = () => (
+  <>
+    <h2 className="text-left text-2xl font-semibold tracking-tight my-3">
+      Agent types
+    </h2>
+    <div className="grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-0 my-6">
+      {types.map((item, index) => {
+        let borderClassName = 'pr-6';
+        if (index % 2 !== 0) borderClassName = ' md:border-l-1.5 pl-6';
```

Comment:
padding is having undesirable effects on mobile (see below)
can use `gap-...` with `grid` or `flex` to seperate elements without retaining padding when responsive

![image](https://github.com/user-attachments/assets/a5a3f819-c19c-40a3-8394-8fae5cf38a19)

Reply:
would it be bad practice if I just added md breakpoints 😅 ?